libtermcap/test_package/conanfile.py

```diff
@@ -11,10 +11,7 @@ def build(self):
         cmake.build()
 
     def test(self):
-        if tools.cross_building(self.settings):
-            self.output.info('This test always fails when cross-compiling. Skipping test.')
-        else:
-            self.run('ctest --output-on-failure')
+        self.run('ctest --output-on-failure')
```

Comment:
Did you really intend to change this file?

Reply:
No. I got confused since every folder has a test_package directory with a conanfile.py in it. I needed to collapse some of those directories in CLion so I wouldn't get confused with what I was actually modifying.